app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -8,6 +8,75 @@ class ManageIQ::Providers::AnsibleTower::Provider < ::Provider
   validates :name, :presence => true, :uniqueness => true
   validates :url,  :presence => true
 
+  PARAMS_FOR_CREATE = {
+    :title  => "Configure Ansible Tower",
+    :fields => [
+      {
+        :component  => "text-field",
+        :name       => "endpoints.default.base_url",
+        :label      => "URL",
+        :isRequired => true,
+        :validate   => [{:type => "required-validator"}]
+      },
+      {
+        :component  => "text-field",
+        :name       => "endpoints.default.username",
+        :label      => "Username",
+        :isRequired => true,
+        :validate   => [{:type => "required-validator"}]
+      },
+      {
+        :component  => "text-field",
+        :name       => "endpoints.default.password",
+        :label      => "Password",
+        :type       => "password",
+        :isRequired => true,
+        :validate   => [{:type => "required-validator"}]
+      },
+      {
+        :component => "checkbox",
+        :name      => "endpoints.default.verify_ssl",
+        :label     => "Verify SSL"
+      }
+    ]
+  }.freeze
+
+  def self.params_for_create
+    PARAMS_FOR_CREATE
+  end
+
+  # Verify Credentials
+  # args:
+  # {
+  #   "endpoints" => {
+  #     "default" => {
+  #       "base_url"   => "",
+  #       "username"   => "",
+  #       "password"   => "",
+  #       "verify_ssl" => ""
+  #     }
+  #   }
+  # }
+  def self.verify_credentials(args)
+    default_endpoint = args.dig("endpoints", "default")
+
+    base_url, username, password, verify_ssl = default_endpoint&.values_at(
+      "base_url", "username", "password", "verify_ssl"
+    )
+
+    !!verify_connection(raw_connect(base_url, username, password, verify_ssl))
```

Comment:
AnsibleTowerClient expects verify_ssl as VERIFY_NONE/VERIFY_PEER not true, false [ref](https://github.com/ansible/ansible_tower_client_ruby/blob/master/lib/ansible_tower_client/connection.rb#L11-L12)
`verify_ssl = verify_ssl ? OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE` ?